Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -174,6 +175,39 @@ void ABenchmarkGymGameModeBase::InitialiseActorCountCheckTimer()
 	}
 }
 
+void ABenchmarkGymGameModeBase::GatherWorkerConfiguration()
+{
+	// No need to fiddle with configuration as the defaults should reflect the native scenario.
```

Comment:
default should reflect the "single server scenario which is all that's required in native" ?